pkg/server/endpoints/endpoints.go

```diff
@@ -110,11 +115,20 @@ func New(ctx context.Context, c Config) (*Endpoints, error) {
 		return entrycache.BuildFromDataStore(ctx, c.Catalog.GetDataStore())
 	}
 
+	updateCacheFn := func(ctx context.Context, cache entrycache.Cache) (err error) {
```

Comment:
is the intention of this feature to go into experimental flag for some time, before making it the default option?